docs/markdown/snippets/has_external_property.md

```diff
@@ -0,0 +1,7 @@
+## Check if native or cross-file properties exist
+
+It is now possible to check whether a native property or a cross-file property
+exists with `meson.has_external_property('foo')`. This is useful if the
+property in question is a boolean and one wants to distinguish between
+"set" and "not provided" which can't be done the usual way by passing a
+fallback parameter to `meson.get_external_property()` in this particular case.
```

Comment:
That's the release snippet, but missing the reference manual doc.

Reply:
True. Added now, thanks!